_translator-files/po/sv-SE/Getting-Started.po

```diff
@@ -200,30 +215,34 @@ msgid ""
 "  <figcaption>Connect to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"  <img src=\"{% include img/sv-SE-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot of the connection window\" style=\"border: 5px solid grey;\">\n"
+"  <figcaption>Anslut till en server</figcaption>\n"
+"</figure>\n"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:71
 #, markdown-text
 msgid "The most important thing about Servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a Server with a ping of less than 50ms if you can."
-msgstr ""
+msgstr "Det viktigaste med servrar är deras \"pingtid\". Ju större siffran är, desto svårare blir det att spela i takt med andra. Vanligtvis bör du välja en server med en ping under 50 ms om du kan."
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:73
 #, markdown-text
 msgid "Once connected to a Server, make sure you can hear yourself OK and fix any input volume or other problems. You can have a look at the [troubleshooting page](/wiki/Client-Troubleshooting) for common problems. And of course, check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
-msgstr ""
+msgstr "När du är ansluten till en server, se till att du kan höra dig själv ordentligt och åtgärda eventuella problem med ingångsvolymen eller andra problem. Du kan titta på [troubleshooting page](/wiki/Client-Troubleshooting) för vanliga problem. Och självklart, kontrollera att du också kan följa [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
```

Comment:
Text in square brackets should be translated and translate text after # in link.

Reply:
`[Den gyllene regeln]` and `ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`